keyboards/clueboard/66/rev4/rev4.c

```diff
@@ -12,35 +11,6 @@ void matrix_scan_kb(void) {
     matrix_scan_user();
 }
 
-void backlight_init_ports(void) {
-    print("init_backlight_pin()\n");
-    // Set our LED pins as output
-    //DDRD |= (1<<6); // Esc
-    //DDRB |= (1<<7); // Page Up
-    //DDRD |= (1<<4); // Arrows
-
-    // Set our LED pins low
-    //PORTD &= ~(1<<6); // Esc
-    //PORTB &= ~(1<<7); // Page Up
-    //PORTD &= ~(1<<4); // Arrows
-}
-
-void backlight_set(uint8_t level) {
-/*
-    if ( level == 0 ) {
-        // Turn off light
-        PORTD |= (1<<6); // Esc
-        PORTB |= (1<<7); // Page Up
-        PORTD |= (1<<4); // Arrows
-    } else {
-        // Turn on light
-        PORTD &= ~(1<<6); // Esc
-        PORTB &= ~(1<<7); // Page Up
-        PORTD &= ~(1<<4); // Arrows
-    }
-*/
-}
-
 void led_init_ports() {
     // Set our LED pins as output
     palSetPadMode(GPIOB, 13, PAL_MODE_OUTPUT_PUSHPULL); // LED1
```

Comment:
These `PUSHPULL` lines can be changed to eg.
```suggestion
    setPinOutput(B13); // LED1
```
And the `palClearPad()`s to `writePinLow()` etc.